frontend/src/_includes/navbar.liquid

```diff
@@ -40,3 +41,12 @@
     </nav>
   </div>
 </header>
+
+<script>
+  let signIn = document.getElementById("signIn");
+  if( window.sessionStorage.getItem("jwt") )
+    signIn.style.display = "none";
```

Comment:
When logged out, this is defaulting to an inline `display: none` for me and not showing the "Sign in" button
 
<img width="563" alt="Screen Shot 2021-04-21 at 2 03 52 AM" src="https://user-images.githubusercontent.com/57154955/115504593-2fe6d700-a246-11eb-8e5e-7018680a59d4.png">

Reply:
weird - not seeing that; I can log out and the sign in button appears;